### DISCUSSION TITLE
Touch Circle Remover

### DISCUSSION THREAD

**Discussion by IdotIcom** [BedrockTweaks/Files#326] [Ideas]
self explanatory.

Will be great for mobile player. (like me)

🙏🙏

**Comment by NSPC911** [comment]
needs shader access

**Reply by IdotIcom** [reply]
So it's not possible??   If so how pack like Better Bedrock and this ↓↓
https://mcpedl.com/touch-circle-remover-1-20/   does it?

**Reply by NSPC911** [reply]
> So it's not possible?? If so how pack like Better Bedrock and this ↓↓ https://mcpedl.com/touch-circle-remover-1-20/ does it?

huh that's weird, I never found that pack before, previous attempts used shaders which arent accessible now, but seems like they are using a different way to do it
